packages/wallets/src/features/cfd/screens/CreatePassword/CreatePassword.tsx

```diff
@@ -7,28 +8,25 @@ import { PlatformDetails } from '../../constants';
 import './CreatePassword.scss';
 
 type TProps = {
-    icon: ReactNode;
     isLoading?: boolean;
     onPasswordChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
     onPrimaryClick: () => void;
     password: string;
     platform: TPlatforms.All;
 };
 
-const CreatePassword: React.FC<TProps> = ({
-    icon,
-    isLoading,
-    onPasswordChange,
-    onPrimaryClick,
-    password,
-    platform,
-}) => {
+const CreatePasswordIcon = {
+    dxtrade: <DerivLightIcDxtradePasswordIcon height={120} width={120} />,
+    mt5: <DerivLightDmt5PasswordIcon height={120} width={120} />,
+} as const;
```

Comment:
Is there a reason why these icons are moved into the CreatePassword component as opposed to being added to their respective components?

Reply:
Sofar at the moment we are just using it in two places which are `DxTradeEnterPasswordModal` and `MT5EnterPasswordModal`. So rather calling the icons in 2 separate files I would rather call the icon within component and make it as an object. In addition we are already, accepting `platforms`, so we can used it as a key to get the icon. 
Another good thing about this is if there's any changes regarding the icon, you only need to change in one place and if there's new icons you can just add it to the object aswell.